pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/filter/query/MultipleMatchQueryStrategy.java

```diff
@@ -5,14 +5,38 @@
 import org.elasticsearch.index.query.QueryBuilder;
 
 public class MultipleMatchQueryStrategy extends BaseQueryStrategy {
+    public static String FIELD_NAME_ANSWER = "activities.questionsAnswers.answer";
+    public static String FIELD_NAME_GROUPED_OPTIONS = "activities.questionsAnswers.groupedOptions.";
 
     @Override
     protected QueryBuilder getMainQueryBuilderFromChild(BaseQueryBuilder baseQueryBuilder) {
         BoolQueryBuilder boolQueryBuilder = new BoolQueryBuilder();
         Object[] values = baseQueryBuilder.payload.getValues();
         for (Object value : values) {
-            boolQueryBuilder.should(new MatchQueryBuilder(baseQueryBuilder.payload.getFieldName(), value));
+            String[] fieldNameAndValue =getCorrectFieldNameAndValues(baseQueryBuilder.payload.getFieldName(), (String) value);
+            boolQueryBuilder.should(new MatchQueryBuilder(fieldNameAndValue[0], fieldNameAndValue[1]));
         }
         return boolQueryBuilder;
     }
+
+    private String[] getCorrectFieldNameAndValues(String fieldName, String value){
```

Comment:
The function returns a String array, size 2, where 0th index contains field name and 1st index contains value name. They are both dependent on each other so one method should return both.